edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudProxy.cs

```diff
@@ -46,8 +46,9 @@ enum Subscriptions
         readonly ResettableTimer timer;
         readonly AsyncLock timerGuard = new AsyncLock();
         readonly bool closeOnIdleTimeout;
-
+        readonly TimeSpan sdkWaitTimeSeconds;
```

Comment:
I know that I said this should have the name "xxSeconds", but that was when it was a numeric value. Now that it is a timespan, I think the name you use to pass this as a parameter is better, and with TimeSpan, the "Seconds" part does not make sense anymore.